doc/nrf/protocols/thread/sed_ssed.rst

```diff
@@ -0,0 +1,64 @@
+
+The polling period can also be configured in runtime.
+See the ``pollperiod`` command in the OpenThread `CLI reference <OpenThread CLI Reference - pollperiod command>`_.
+
+Synchronized Sleepy End Device
+******************************
+
+SSEDs are MTDs that are further optimized for power consumption.
+A Thread SSED is synchronized with its parent Router and uses the radio only at scheduled intervals.
+It does this by using the :ref:`thread_ug_supported_features_csl` feature (introduced as one of the `Thread 1.2 Base Features`_).
+During those intervals, the device waits for the Router to send it any data related to the desired device activity.
+This reduces the network traffic (since there is no polling) and the power consumption (since the radio is off most of the time).
+
+An SSED does require sending packets occasionally to keep synchronization with the Router.
+However, unlike a regular SED, an SSED does not actively communicate with the Router by polling and it goes into the idle mode between the scheduled activity periods.
+If there is no application-related traffic for an extended period of time, the SSED sends a data poll request packet to synchronize with the parent.
+
+The |NCS| provides the following Kconfig options that let you enable CSL and specify the CSL parameters:
+
+* :kconfig:option:`CONFIG_OPENTHREAD_CSL_RECEIVER` - Enables SSED child mode.
+* :kconfig:option:`CONFIG_OPENTHREAD_CSL_AUTO_SYNC` - Enables the CSL autosynchronization feature.
+* :kconfig:option:`CONFIG_OPENTHREAD_CSL_TIMEOUT` - Sets the default CSL timeout in seconds.
+* :kconfig:option:`CONFIG_OPENTHREAD_CSL_CHANNEL` - Sets the default CSL channel.
+  This option corresponds to the ``csl channel`` CLI parameter.
+
+The following Kconfig options affect the size of the receive window, and thus also affect the device's power consumption:
+
+* :kconfig:option:`CONFIG_OPENTHREAD_CSL_RECEIVE_TIME_AHEAD` - Sets the CSL receiver wake up margin in microseconds.
+* :kconfig:option:`CONFIG_OPENTHREAD_MIN_RECEIVE_ON_AHEAD` - Sets the minimum receiving time before start of MAC header.
+* :kconfig:option:`CONFIG_OPENTHREAD_MIN_RECEIVE_ON_AFTER` - Sets the minimum receiving time after start of MAC header.
+* :kconfig:option:`CONFIG_OPENTHREAD_PLATFORM_CSL_UNCERT` - Sets the fixed uncertainty of the device for scheduling CSL Transmissions in units of 10 microseconds.
+
+Additionally, you must configure the ``period`` CLI parameter to enable CSL.
+For more information on using the CLI to configure parameters for CSL, see the ``csl`` command in the OpenThread `CLI reference <OpenThread CLI Reference - csl command_>`_.
+
+Comparison of SED and SSED
+**************************
+
+Compared to an SED, an SSED has no drawbacks for transmission and provides reduced power consumption and network traffic.
+This means you should configure your SED devices as SSEDs whenever possible.
```

Comment:
I would definitely not advise to use SSED always but only for cases where we need a small latency (small csl interval) and want to have similar or smaller power consumption than SED has (e.g. window coverings). For sensors that report a new measurement each e.g. 30-60 seconds, is it justified to use CSL?

The common misconception is that SSED role is just better version of SED role. This is not true, it was designed for specific use cases and I would still expect vast majority of devices to use SED role rather than SSED role. 

@edmont could you comment here? I would like to provide a clear recommendation when usage of SSED is better. Unfortunately, we need to account the possibility of being a child of a router that has very large uncertainty - which would make solution much worse than SED.

Reply:
In general e.g. from HomeKit or Matter perspective, it would be great that potential user will read this section and will be able to choose the SED or SSED role depending on its device type / communication patters.

Reply:
I totally agree, SSEDs are not intended to fully replace SEDs. As a rule of thumb it would be beneficial (in terms of energy and spectrum usage) to used SSEDs when the application requires to poll more often than 20 seconds. But this depends on many factors, like routers CSL Accuracy. KRKNWK-13546 should address this topic more in detail.